components/console-backend-service/internal/domain/servicecatalogaddons/clusteraddonsconfiguration_resolver.go

```diff
@@ -100,8 +112,14 @@ func (r *clusterAddonsConfigurationResolver) DeleteClusterAddonsConfiguration(ct
 	return r.addonsCfgConverter.ToGQL(addon), nil
 }
 
-func (r clusterAddonsConfigurationResolver) AddClusterAddonsConfigurationURLs(ctx context.Context, name string, urls []string) (*gqlschema.AddonsConfiguration, error) {
-	addon, err := r.addonsCfgUpdater.AddRepos(name, urls)
+func (r clusterAddonsConfigurationResolver) AddClusterAddonsConfigurationURLs(ctx context.Context, name string, repositories []gqlschema.AddonsConfigurationRepositoryInput, urls []string) (*gqlschema.AddonsConfiguration, error) {
+	repositories, err := resolveRepositories(repositories, urls)
```

Comment:
if both fields are set (urls and repositories) the error should be readable in the graphQL client